Samples/Server/Unity-FractalStream/Assets/Scripts/FractalRoot.cs

```diff
@@ -1,13 +1,9 @@
-﻿using System.Collections;
-using System.Collections.Generic;
-using UnityEngine;
+﻿using UnityEngine;
 
-
-
-public class FractalRoot : MonoBehaviour {
+public class FractalRoot : MonoBehaviour
+{
 
     Fractals rootSettings;
-    VoiceManager voiceManager;
```

Comment:
pulled out the voice manager bc i don't think it worked, and isn't particularly useful in a basic sample anyway. if we want that we should write a different sample (or scene) imo